VARIABLES.md

```diff
@@ -1838,7 +1870,7 @@ Below are the supported variables for the role confluent.zookeeper
 
 ### zookeeper_custom_log4j
 
-Boolean to enable cp-ansible's Custom Log4j Configuration
+Boolean to reconfigure Zookeeper's logging with RollingFileAppender and log cleanup
```

Comment:
Boolean to reconfigure Zookeeper's logging with the RollingFileAppender and log cleanup functionality.